core-bundle/contao/themes/flexible/styles/main.css

```diff
@@ -2752,7 +2752,13 @@ label.error, legend.error, .tl_checkbox_container.error legend {
 }
 
 /* Fieldsets */
+fieldset legend {
+	padding-inline-start: 0;
+	padding-inline-end: 0;
```

Comment:
What does that do? Unsetting these rules does not make a difference on my system.

Reply:
Resetting the default behavior of the browser itself:
![image](https://github.com/user-attachments/assets/12ead102-2009-4e04-aea7-5b0c24e2fad6)

I think it's overwritten by the following line and could be removed
https://github.com/contao/contao/blob/52fa48dd74dbf2ea528f6d0471c6f43cc3155ecd/core-bundle/contao/themes/flexible/styles/main.css#L2771

Reply:
@leofeyer iirc, this must have been before I used a full button but I'm unsure. Removing it did not change anything.

Reply:
Then let’s remove it again and see what happens 😄

Reply:
Already removed in https://github.com/contao/contao/pull/7235/commits/1a803e37036325ea876ff4c8e727b6f8a37f55d5 like 3hrs ago ;)